linera-storage/src/lib.rs

```diff
@@ -339,6 +349,16 @@ pub trait Storage: Sized {
     ) -> Result<Self::BlockExporterContext, ViewError>;
 }
 
+/// A description of the current Linera network to be stored in every node's database.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[cfg_attr(with_testing, derive(Eq, PartialEq))]
+pub struct NetworkDescription {
+    pub name: String,
+    pub genesis_config_hash: CryptoHash,
+    pub timestamp: Timestamp,
```

Comment:
```suggestion
    pub genesis_timestamp: Timestamp,
```
Then it becomes self-explanatory like the rest of the fields.

Reply:
yes good idea